Shifty.App/Shared/NavMenu.razor

```diff
@@ -1,4 +1,20 @@
 ﻿<MudNavMenu>
     <MudNavLink Href="" Match="NavLinkMatch.All" Icon="@Icons.Material.Filled.Home">Home</MudNavLink>
     <MudNavLink Href="ShiftsCheckIn" Match="NavLinkMatch.Prefix" Icon="@Icons.Material.Filled.MeetingRoom">Shift Check-in</MudNavLink>
```

Comment:
You may want to remove this one as well, since it no longer points anywhere

Reply:
If we are not to use the page at all, we need to remove all references to it, right? Not just removing it from the navigation menu and if so, then maybe it should be a small PR by itself?